dissect/target/plugin.py

```diff
@@ -527,7 +528,7 @@ def decorator(obj):
     return decorator
 
 
-def plugins(osfilter: str = None) -> Iterator[PluginDescriptor]:
+def plugins(osfilter: Type[OSPlugin] = None) -> Iterator[PluginDescriptor]:
```

Comment:
```suggestion
def plugins(osfilter: type[OSPlugin] = None) -> Iterator[PluginDescriptor]:
```
Or is that not allowed?